.github/workflows/check-spelling.yml

```diff
@@ -10,4 +10,6 @@ on:
 jobs:
   check-spelling:
     name: Check Spelling
-    uses: Lombiq/GitHub-Actions/.github/workflows/spelling.yml@dev
+    uses: Lombiq/GitHub-Actions/.github/workflows/spelling.yml@OSOE-743
+    with:
+      cancel-in-progress-for-this-pr: 'true'
```

Comment:
I'd use this as the default, see my comment under https://github.com/Lombiq/GitHub-Actions/pull/290.